src/operator/softmax_output.cc

```diff
@@ -62,7 +62,7 @@ MXNET_REGISTER_OP_PROPERTY(SoftmaxOutput, SoftmaxOutputProp)
 
     .. math:: \text{CE(label, output)} = - \sum_i \text{label}_i \log(\text{output}_i)
 
-  - The gradient of cross entropy loss w.r.t softmax output:
+  - The gradient of cross entropy loss w.r.t the input array:
 
     .. math:: \text{gradient} = \text{output} - \text{label}
```

Comment:
This is indeed describing the loss w.r.t. the output, not input

Reply:
As I've searched, the gradient will give a correction(i.e. gradient) which can be represented by the softmax activation to the features before softmax output(activation), not the softmax output(activation).
https://math.stackexchange.com/questions/945871/derivative-of-softmax-loss-function

Reply:
@piiswrong  As @chinakook  said, I think it is the gradient of the loss wrt the input of the softmax activation function